docs/sources/shared/visualizations/tooltip-options-1.md

```diff
@@ -11,3 +11,15 @@ When you hover your cursor over the visualization, Grafana can display tooltips.
 - **Hidden -** Do not display the tooltip when you interact with the visualization.
 
 Use an override to hide individual series from the tooltip.
+
+### Values sort order
+
+When you set the **Tooltip mode** to **All**, the **Values sort order** option is displayed. This option controls the order in which values are listed in a tooltip. Choose from the following:
+
+- **None** - Grafana automatically sorts the values displayed in a tooltip.
+- **Ascending** - Values in the tooltip are listed from smallest to largest.
```

Comment:
`from smallest to largest` - is there a better/broader word we could use here? Feels (to me) like we're just referring to numbers (we can also sort strings). But no strong opinion 😁